src/org/omegat/gui/editor/mark/ComesFromMTMarker.java

```diff
@@ -73,7 +73,7 @@ public void setMark(SourceTextEntry ste, String text) {
     public synchronized List<Mark> getMarksForEntry(SourceTextEntry ste, String sourceText, String translationText,
             boolean isActive) {
         synchronized (this) {
-            if (!isActive || ste != markedSte || !translationText.equals(markedText)) {
```

Comment:
Marker is activated  when

1. active edit
2. source entry is registered  as come from tm/mt
3. translation is not null
4. translation is registered as come from tm/mt